python/dgl/graphbolt/impl/temporal_neighbor_sampler.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         self.sampler = graph.temporal_sample_neighbors
 
     def sample_subgraphs(self, seeds, seeds_timestamp=None):
+        assert seeds_timestamp is not None, "seeds_timestamp must be provided."
```

Comment:
can we remove the default value in above line as it looks weird that `seeds_timestamp` is None in default but it shouldn't be None.

Reply:
I find it's the same method signature of base `SubgraphSampler`. But such signature has to be for `NeighborSampler` looks weird as it seems such additional argument is for `TemporalNS` only.

Reply:
Make sense. I removed the default value.